README.md

```diff
@@ -1,37 +1,60 @@
-</div>
-<br><br>
+To train PointNet:
 
+`python run.py experiment=PointNet_baseline`
 
-Myria3D is a deep learning library designed with a focused scope: the multiclass semantic segmentation of large scale, high density aerial Lidar points cloud.
 
-The library implements the training of 3D Segmentation neural networks, with optimized data-processing and evaluation logics at fit time. Inference on unseen, large scale point cloud is also supported.
-It allows for the evaluation of single-class IoU on the full point cloud, which results in reliable model evaluation.
+To train RandLaNet:
+
+`python run.py experiment=RandLaNet_base_run_FR`
 
-Myria3D is built upon [PyTorch](https://pytorch.org/). It keeps the standard data format 
-from [Pytorch-Geometric](https://pytorch-geometric.readthedocs.io/). 
-Its structure was bootstraped from [this code template](https://github.com/ashleve/lightning-hydra-template),
-which heavily relies on [Hydra](https://hydra.cc/) and [Pytorch-Lightning](https://github.com/PyTorchLightning/pytorch-lightning) to enable flexible and rapid iterations of deep learning experiments.
 
-Although the library can be extended with new neural network architectures or new data signatures, it makes some opiniated choices in terms of neural network architecture, data processing logics, and inference logic. Indeed, it is initially built with the [French "Lidar HD" project](https://geoservices.ign.fr/lidarhd) in mind, with the ambition to map France in 3D with 10 pulse/m² aerial Lidar by 2025. The data will be openly available, including a semantic segmentation with a minimal number of classes: ground, vegetation, buildings, vehicles, bridges, others. 
+To infer PointNet model:
 
-> &rarr; For installation and usage, please refer to [**Documentation**](https://ignf.github.io/myria3d/).
+`python run.py task.task_name='predict' predict.src_las='/path/to/las' predict.output_dir='/path/to/output' datamodule.epsg=2154 predict.ckpt_path='${hydra:runtime.cwd}/trained_model_assets/randlanet_norgb_epoch_028.ckpt' trainer.accelerator=gpu predict.gpus=[0]`
 
-> &rarr; A stable, production-ready version of Myria3D is tracked by a [Production Release](https://github.com/IGNF/myria3d/releases/tag/prod-release-tag). In the release's assets are a trained multiclass segmentation model as well as the necessary configuration file to perform inference on French "Lidar HD" data. Those assets are provided for convenience, and are subject to change in time to reflect latest model training.
+(to achieve better results add `predict.subtile_overlap=25`)
+
+
+To infer RandLaNet model:
+
+`python run.py task.task_name='predict' predict.src_las='/path/to/las' predict.output_dir='/path/to/output' datamodule.epsg=2154 predict.ckpt_path='${hydra:runtime.cwd}/trained_model_assets/pointnet_norgb_epoch_020.ckpt' trainer.accelerator=gpu predict.gpus=[0]`
 ___
 
+# Comparisons
```

Comment:
I think that your comparisons can have a place in the documentation too, provided that you update the links so that the images are stored in our repo as well (in case yours is modified)